var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -73,6 +74,10 @@ def install(self, spec, prefix):
             # For @develop + use elfutils libdw, libelf is an abstraction
             # we are really using elfutils here
             if spec.satisfies('@develop'):
+                tbb = spec['tbb'].prefix
+                args.append('-DTBB_INCLUDE_DIRS=%s' % tbb.include)
+                args.append('-DTBB_LIBRARIES=%s'   % join_path(
```

Comment:
this is very picky, but it looks like you are one space off from lining up your '%' characters